### DISCUSSION TITLE
Translation contributions

### DISCUSSION THREAD

**Discussion by markcst** [KRTirtho/spotube#911] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

HI, @KRTirtho !
How can I contribute to the app translation?
I'm using the deb package on Mint, and I'd like to write Italian localization for it

### Suggested solution

Add a button to send you feature requests through the app
Add a menu voice (in the app language selection dropdown) for add our translation

**Comment by KRTirtho** [comment]
Check the [CONTRIBUTION.md](https://github.com/KRTirtho/spotube/blob/master/CONTRIBUTION.md#submit-translations)
